labels.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import requests
+import json
+from webcompat import app
+
+API_URI = 'https://api.github.com/repos/'
+ISSUES_URI = app.config['ISSUES_REPO_URI'].rpartition('/issues')[0]
+USER_LABELS_URI = API_URI + ISSUES_URI + "/labels"
+WEBCOMPAT_LABELS_URI = API_URI + 'webcompat/webcompat-tests/labels'
+AUTH_HEADERS = {'Authorization': 'token {0}'.format(app.config['OAUTH_TOKEN']),
+                'User-Agent': 'webcompat/webcompat-bot'}
+
+
+def get_issue_labels(labels_uri):
+    '''Get the issue labels from the given repo.'''
+    r = requests.get(labels_uri)
+    return r.json()
+
+
+def create_label(data):
+    '''Creates a label in the given repo'''
+    requests.post(USER_LABELS_URI, json.dumps(data), headers=AUTH_HEADERS)
```

Comment:
@dshgna You might probably want to forbid the create_label when the repo is `webcompat/webcompat-tests` and `webcompat/webcompat.com`. So a user with the right privilege doesn't by accident erase and re-create the labels. Typos and sleepless nights are so quick.

Maybe to be less magic, I would probably give `def create_label(labels, repo_uri)` in the spirit of `what` and `where`

Also it would be good to test the result of your request here and return a code/string for the user to know if it really happened. try/except probably.